### PR TITLE
T5260: Do not use deprecated python crypt module

### PR DESCRIPTION
**Pull request by cuongdt1994**
## Change Summary

Needed for PR: 
https://vyos.dev/T5260
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)

* https://vyos.dev/T5620

## Proposed changes

Remove crypt and change to passlib
## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [ ] I have linked this PR to one or more Phabricator Task(s)
- [ ] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [ ] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Comment by sever-sever** [comment]
@cuongdt1994  You should reference the Task id in your commit message.
For example, `T5260: Do not use deprecated python crypt module`
Thanks.

File: src/conf_mode/vpn_openconnect.py [hunk 1]
```diff
@@ -45,7 +45,8 @@
 
 # Generate hash from user cleartext password
 def get_hash(password):
-    return crypt(password, mksalt(METHOD_SHA512))
+    return sha512_crypt.hash("password")
```
Comment (sarthurdev) [inline src/conf_mode/vpn_openconnect.py]: Remove the quotes, this will always return the hash for string "password".

**Review by sarthurdev — CHANGES_REQUESTED** [review]

**Comment by sever-sever** [comment]
The commit message is still the same. You changed only title.
commit

File: src/conf_mode/vpn_openconnect.py (hunk 1 above)
Comment (cuongdt1994) [reply src/conf_mode/vpn_openconnect.py]: Resolved, thank for your comment!

**Comment by cuongdt1994** [comment]
Changed the commit message!

image

**Comment by sever-sever** [comment]
@cuongdt1994 Thanks!

**Review by sever-sever — APPROVED** [review]

**Pull request merged**